joark/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/joark/dokument/PdfDokument.kt

```diff
@@ -2,43 +2,41 @@
 
 package no.nav.helsearbeidsgiver.inntektsmelding.joark.dokument
 
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.AarsakInnsending
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Bonus
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Feilregistrert
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Ferie
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Ferietrekk
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Inntektsmelding
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.NyStilling
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.NyStillingsprosent
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Nyansatt
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Permisjon
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Permittering
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Sykefravaer
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.Tariffendring
-import no.nav.helsearbeidsgiver.domene.inntektsmelding.deprecated.VarigLonnsendring
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.AarsakInnsending
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Bonus
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Feilregistrert
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Ferie
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Ferietrekk
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Inntektsmelding
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.NyStilling
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.NyStillingsprosent
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Nyansatt
 import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Periode
-import java.time.LocalDate
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Permisjon
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Permittering
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Sykefravaer
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Tariffendring
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.VarigLoennsendring
 
 private const val FORKLARING_ENDRING = "Forklaring for endring"
 
 class PdfDokument(
-    val dokument: Inntektsmelding,
+    val inntektsmelding: Inntektsmelding,
 ) {
-    private val pdf = PdfBuilder(bodySize = 20, topText = "Innsendt: ${dokument.tidspunkt.toNorsk()}") // Setter skriftstørrelsen på labels og text
+    private val pdf = PdfBuilder(bodySize = 20, topText = "Innsendt: ${inntektsmelding.mottatt.toNorsk()}") // Setter skriftstørrelsen på labels og text
     private var y = 0
     private val kolonneEn = 0
     private val kolonneTo = 420
     private val naturalytelse1 = kolonneEn
     private val naturalytelse2 = kolonneEn + 400
     private val naturalytelse3 = kolonneEn + 700
```

Comment:
Kanskje den gjengen her kunne vært `private const val` og i stor bokstaver og snake case?

Reply:
`const` er ikke lov for klassefelt 😞